ruby/Rakefile

```diff
@@ -6,5 +6,6 @@ file 'lib/cucumber/ci_environment/CiEnvironments.json' => '../CiEnvironments.jso
   FileUtils.cp t.prerequisites[0], t.name
 end
 
+desc 'Populates directory with test file'
```

Comment:
This isn't strictly speaking true.

When using rake, any task provided beforehand is a pre-requisite. This job here is running the tests, but before it runs the tests it requires a pre-requisite.

The `spec` task is pre-provided from rspec rake.

So a more apt name here would be something like

> Copy over environments metadata and run tests